### PR TITLE
initial commit farsi (Iran) spelling dictionary in UTF-8

### PR DESCRIPTION
**Pull request by AhmadJavadiNejad**
hi
Please accept my request to add Persian Dictionary.
be soon the rest of the sections will be completed.
Thanks in advance

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,7 @@ archive/
 source/
 node_modules/
 yarn.lock
+/.vs/slnx.sqlite
```
Comment (wooorm) [inline .gitignore]: Could you remove these patterns?

Comment (AhmadJavadiNejad) [reply .gitignore]: yes,sure I'll delete it.

**Comment by AhmadJavadiNejad** [comment]
> Thanks for working on this, @AhmadJavadiNejad!
> 
> The main question I have is where the source of the dictionary comes from: did you write it yourself, or did you get the dictionary from somewhere else?
> If the former, I have some other questions, but I suspect the latter, in which case, I can’t accept this PR because it does not reference where it comes from, and that’s probably a breach of contract / license.

Yes sure. in the next commit, I'll provide references for the dictionary.

**Comment by AhmadJavadiNejad** [comment]
> Thanks for working on this, @AhmadJavadiNejad!
> 
> The main question I have is where the source of the dictionary comes from: did you write it yourself, or did you get the dictionary from somewhere else?
> If the former, I have some other questions, but I suspect the latter, in which case, I can’t accept this PR because it does not reference where it comes from, and that’s probably a breach of contract / license.

hi
the [license and vocabulary sources](https://github.com/wooorm/dictionaries/pull/10/commits/1a99ae33eac7e8b33fb5dade14b458a5d55bb874) are specified.
thanks.

**Comment by wooorm** [comment]
Any reason why you’re not basing this off [b00f/lilak](https://github.com/b00f/lilak)? That looks better as it has a working .aff file.

**Comment by AhmadJavadiNejad** [comment]
> Any reason why you’re not basing this off [b00f/lilak](https://github.com/b00f/lilak)? That looks better as it has a working .aff file.

There is no particular reason, so far I did not see this repository. I was also looking to Writing a .affx file.

**Pull request closed without merging**